packages/tools/package/src/commands/changelog/changelog.utils.ts

```diff
@@ -54,7 +54,7 @@ export async function createOrUpdateChangelog(packages: Package[], prevCommitId:
     if (version && commits) {
       changelogEntry[pkgName] = {
         [version]: {
-          published_date: Math.floor(Date.now() / 1000),
+          published_date: Math.floor(Date.now()),
```

Comment:
How does this fix the date?

Reply:
When we do Date.now() It returns the number of seconds elapsed since midnight, January 1, 1970 Universal Coordinated Time (UTC). Because I was dividing it by 1000 the value was reduced hence the date moved to a past date.